frontend/src/metabase-lib/v1/expressions/resolver.js

```diff
@@ -89,6 +89,12 @@ export function resolve({
     const [op, ...operands] = expression;
 
     if (op === "value") {
+      const [value] = operands;
```

Comment:
As of https://github.com/metabase/metabase/pull/55737 this is obsolete and we can just rely on the type checking in diagnose-expression. I will merge in the latest master and revert this and resolve conflicts in the tests.